tcms/issuetracker/types.py

```diff
@@ -364,3 +369,61 @@ def report_issue_from_testcase(self, caserun):
             url += '/'
 
         return url + '/issues/new?' + urlencode(args, True)
+
+
+class Gitlab(IssueTrackerType):
+    """
+        Support for Gitlab. Requires:
+
+        :base_url: - URL to a Gitlab repository for which we're going to report issues
+        :api_password: - Gitlab API token.
+    """
+
+    def __init__(self, tracker):
+        super(Gitlab, self).__init__(tracker)
+
+        # NOTE: we use an access token so only the password field is required
```

Comment:
Remove the 'NOTE' part b/c this generates unnecessary warnings with pylint. Leave the rest of the comment.